frontend/src/scenes/billing/UnsubscribeSurveyModal.tsx

```diff
@@ -71,46 +82,40 @@ export const UnsubscribeSurveyModal = ({
                         setSurveyResponse(value, '$survey_response')
                     }}
                 />
-                <LemonBanner type="info">
-                    <p>
-                        {'Need to control your costs? Learn about ways to '}
-                        <Link
-                            to="https://posthog.com/docs/billing/estimating-usage-costs#how-to-reduce-your-posthog-costs"
-                            target="_blank"
-                            onClick={() => {
-                                reportSurveyDismissed(surveyID)
-                            }}
-                        >
-                            reduce your bill
-                        </Link>
-                        {`${product.type !== 'session_replay' ? ' or ' : ', '}`}
-                        <Link
-                            to="mailto:sales@posthog.com?subject=Help%20reducing%20PostHog%20bill"
-                            target="_blank"
-                            onClick={() => {
-                                reportSurveyDismissed(surveyID)
-                            }}
-                        >
-                            chat with support
-                        </Link>
-                        {product.type === 'session_replay' && (
-                            <>
-                                {', or '}
-                                <Link
-                                    to="mailto:sales@posthog.com?subject=Joining%session%replay%controls%20beta"
-                                    target="_blank"
-                                    onClick={() => {
-                                        reportSurveyDismissed(surveyID)
-                                    }}
-                                >
-                                    join our beta
-                                </Link>
-                                {' for tuning recording volume with sampling and minimum duration.'}
-                            </>
-                        )}
-                        .
-                    </p>
-                </LemonBanner>
+                <LemonCollapse
```

Comment:
I don't like this as a collapse, it will get even fewer clicks IMO and it looks kinda weird. 

What if we turn this back into an info banner, and put the payment info banner below it. It'll have two info banners, which I think will look a little funny but might be okay.

Reply:
There's not that [many people clicking](https://us.posthog.com/project/2/insights/9GJX2xIi) on either of the alternative options. I'm not sure it matters if we hide it any further. 

Even if we keep them both as info level banners, I think it makes sense to put the one about auto-paying invoices above the survey text area. People read from top-to-bottom, so they should see that first and exit out if necessary. I also think we should keep the link to the stripe page in the banner.

Reply:
I want some kind of distinction between the two banners and think stacking them looks quite bad. I think, at a minimum, having them sandwich the text area will be ok.